src/actions/users.js

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+export const USER_CREATED = "USER_CREATED";
+export const LOGIN_SUCCESS = "LOGIN_SUCCESS";
+
+const dbUrl = process.DATABASE_URL || "http://localhost:4000";
```

Comment:
this maybe belongs in a constant file somewhere so you don't define it multiple times in each action file